src/patternfly/components/Button/styles.scss

```diff
@@ -119,49 +119,29 @@
   text-align: center;
   white-space: nowrap;
   background-color: var(--pf-c-button--BackgroundColor);
-  border: var(--pf-c-button--BorderWidth) solid var(--pf-c-button--BorderColor);
+  border: 0;
   border-radius: var(--pf-c-button--BorderRadius);
+  box-shadow: inset 0 0 0 var(--pf-c-button--BoxShadowSpread) var(--pf-c-button--BoxShadowColor);
```

Comment:
I had defined the [variable names for dropdown](https://github.com/patternfly/patternfly-next/blob/master/src/patternfly/components/Dropdown/styles.scss#L63) differently, but I actually think this is correct. I was using 'Border' instead of 'BoxShadow' which is conceptually what you're affecting, but really I think we're supposed to map names to the actual selector.

@andresgalante can you confirm this?

Reply:
@jgiardino I agree, and @michael-coker's naming convention follows the spec exactly. My thinking is this. We're mimicking border with box-shadow (kind of a hack, I'd like to say workaround) in order to avoid adjusting height, because of border-width changes on hover and focus. But, since we can add multiple drop-shadows, I like the idea of describing the border and box-shadow vals discretely if, in the future, we need to add a traditional shadow to button (seems to be an emerging design pattern). However, I'm def ok w/defining the vars this way.. @andresgalante your thoughts?

Reply:
Also, can we add "outline: none;" here? Not part of the request, but would be beneficial.

Reply:
@mattnolting done!